src/openrct2/game.c

```diff
@@ -1121,6 +1121,22 @@ void game_fix_save_vars()
 
     // Fix invalid vehicle sprite sizes, thus preventing visual corruption of sprites
     fix_invalid_vehicle_sprite_sizes();
+
+    // Fix gEntrance locations for which the map_element no longer exists
+    uint8 entranceNum;
```

Comment:
this variable should be declared in the `for` loop.

Reply:
Done.